internal/entitlement/connector.go

```diff
@@ -120,6 +128,24 @@ func (c *entitlementConnector) CreateEntitlement(ctx context.Context, input Crea
 			return nil, err
 		}
 
+		event, err := eventtypes.NewCloudEvent(
```

Comment:
@chrisgacsal This is the expected use of the publishing. I have added this TopicPublisher to the publisher interface as I don't think it would make sense for the connector to actually be aware of the target topic.